packages/slate-dev-logger/src/index.js

```diff
@@ -82,11 +89,12 @@ function warn(message, ...args) {
  */
 
 function deprecate(version, message, ...args) {
-  if (FORBID_DEPRECATE) {
-    throw new Error(`Deprecation (${version}): ${message}`)
-  }
+  const logger = FORBID_DEPRECATE ? forbidden : log
+  logger('warn', `Deprecation (${version}): ${message}`, ...args)
+}
 
-  log('warn', `Deprecation (${version}): ${message}`, ...args)
+function forbidden(level, message, ...args) {
```

Comment:
Can you add a doc block comment here? I think we should remove the `...args` as well, since they will make the error message strange I think.

Reply:
Agree.  adding args will make Error quite strange.  Fixed~